### PR TITLE
Deep sleep tutorial.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/hardware/deepsleep.mdx [hunk 1]
```diff
@@ -1 +1,136 @@
...
+<Note>
+
+If you are using [Toit's fleet management](../../getstarted/fleet), then
+you don't need to worry about deep sleep. Artemis (the installed
+container of the fleet management system) automatically puts the device
+into deep sleep when it is not running any programs.
+
+</Note>
+
+The ESP32 touch sensor detects variations in capacitance on the GPIO pins,
```
Comment (kasperl) [inline docs/tutorials/hardware/deepsleep.mdx:14]: wat?

Should this go elsewhere?

File: docs/tutorials/hardware/deepsleep.mdx [hunk 2]
```diff
@@ -1 +1,136 @@
...
+  touch.threshold = 800
+  esp32.enable_touchpad_wakeup
+  esp32.deep_sleep (Duration --m=3)
+```
+
+The ESP32 will go to sleep for three minutes, or wake up if it
+registers a touch on pin 32.
+
+## Power consumption
+When the ESP32 is in deep sleep it only consumes about 10µA. However,
```
Comment (kasperl) [inline docs/tutorials/hardware/deepsleep.mdx:131]: I think it is actually less than that.

File: docs/tutorials/hardware/deepsleep.mdx [hunk 3]
```diff
@@ -1 +1,136 @@
...
+this assumes that no other components are drawing any power. There is
+a significant difference between different ESP32 boards.
```
Comment (kasperl) [inline docs/tutorials/hardware/deepsleep.mdx:133]:
```suggestion
this assumes that no other components are drawing any power. There are
significant differences between different ESP32 boards.
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/tutorials/hardware/deepsleep.mdx (hunk 2 above)
Comment (floitsch) [reply docs/tutorials/hardware/deepsleep.mdx:128]: Actually went to the datasheet, and there it is 10.

**Pull request merged**